feature/challenge-reward/data/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+  id("appcoins.android.library")
+}
+
+android {
+  namespace = "com.appcoins.wallet.feature.challengereward.data"
+
+  @Suppress("UnstableApiUsage")
+  buildFeatures {
+    compose = true
+  }
+
+  @Suppress("UnstableApiUsage")
+  composeOptions {
+    kotlinCompilerExtensionVersion = "1.4.4"
+  }
+}
+
+dependencies {
+  implementation(project(":legacy:billing"))
+  implementation(project(":core:analytics"))
+  implementation(project(":core:network:microservices"))
+  implementation(project(":feature:wallet-info:data"))
+
+  // FYBER
+  implementation("com.fyber:fairbid-sdk:3.42.0")
```

Comment:
we have the file libs.version to do that, please move it, and you can import the rxjava bundle